config/artsey.dtsi

```diff
@@ -142,7 +145,7 @@
  * Define custom layer
  *
  * You can override this layer by copy/pasting the into your keymap *before* including the artsey.dtsi file
- * Once added to your keymap you can adjust as you see fit
+ * Once added to your keymap you can adjust as you see fit.
```

Comment:
Please remove the period. It's outside standard comment conventions in this file.